### ISSUE TITLE
`debugChannel.unsubscribe` is not a function.

### ISSUE THREAD

**Issue by NoorShamasneh** [DataDog/dd-trace-js#2768]
I get this error when I try to run  `NODE_OPTIONS="-r dd-trace/ci/init" jest` 👇 
```
/.../node_modules/dd-trace/packages/dd-trace/src/log/writer.js:26
  debugChannel.unsubscribe(onDebug)
               ^

TypeError: debugChannel.unsubscribe is not a function
    at unsubscribeAll (/.../node_modules/dd-trace/packages/dd-trace/src/log/writer.js:26:16)
    at toggleSubscription (/.../node_modules/dd-trace/packages/dd-trace/src/log/writer.js:33:3)
    at Object.reset (/.../node_modules/dd-trace/packages/dd-trace/src/log/writer.js:69:3)
    at Object.reset (/.../node_modules/dd-trace/packages/dd-trace/src/log/index.js:35:15)
    at Object.<anonymous> (/.../node_modules/dd-trace/packages/dd-trace/src/log/index.js:77:5)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)

```

**Environment**

* **Operation system: macOS 13.1**
* **Node.js version: 16.13.1**
* **Tracer version: 3.13.1**
* **Relevant library versions:**

**Comment by mtorre4580-peya** [comment]
I have the same problem with the latest version

**Comment by magnonta** [comment]
I have same issue here:

```
2/3/2023, 5:06:28 PM | error Command failed with exit code 1. | node
-- | -- | --
2/3/2023, 5:06:28 PM | /usr/src/app/node_modules/dd-trace/packages/dd-trace/src/log/writer.js:26 | node
2/3/2023, 5:06:28 PM | debugChannel.unsubscribe(onDebug) | node
2/3/2023, 5:06:28 PM | ^ | node
2/3/2023, 5:06:28 PM | TypeError: debugChannel.unsubscribe is not a function | node
2/3/2023, 5:06:28 PM | at unsubscribeAll (/usr/src/app/node_modules/dd-trace/packages/dd-trace/src/log/writer.js:26:16) | node
2/3/2023, 5:06:28 PM | at toggleSubscription (/usr/src/app/node_modules/dd-trace/packages/dd-trace/src/log/writer.js:33:3) | node
2/3/2023, 5:06:28 PM | at Object.reset (/usr/src/app/node_modules/dd-trace/packages/dd-trace/src/log/writer.js:69:3) | node
2/3/2023, 5:06:28 PM | at Object.reset (/usr/src/app/node_modules/dd-trace/packages/dd-trace/src/log/index.js:35:15) | node
2/3/2023, 5:06:28 PM | at Object.<anonymous> (/usr/src/app/node_modules/dd-trace/packages/dd-trace/src/log/index.js:77:5)
```

**Comment by tpickett66** [comment]
I just ran into this on Node 14 when trying to use both 3.13.1 and 3.13.0

**Comment by ParikshitChavan** [comment]
Upgrading to version `3.13.2` should fix this issue for you for you Since the bug has [been fixed here](https://github.com/DataDog/dd-trace-js/pull/2770) and released in the [patch version `3.13.2`](https://github.com/DataDog/dd-trace-js/releases/tag/v3.13.2)

**Comment by simon-id** [comment]
Hello everyone and thank you for the bug report.
As @ParikshitChavan mentionned, we've fixed this bug in versions [3.13.2](https://github.com/DataDog/dd-trace-js/releases/tag/v3.13.2) and [2.26.2](https://github.com/DataDog/dd-trace-js/releases/tag/v2.26.2) of `dd-trace`.
To be clear: the bug was only affecting outdated versions of Node.js (`>=14.17.0/15.1.0 <14.19.0/16.14.0/17.1.0`) as the bug stemmed from Node.js itself.

**Comment by NoorShamasneh** [comment]
Thanks @simon-id.
